volume/volume_test.go

```diff
@@ -1,11 +1,46 @@
+// +build !volumecreate
+
 package volume
 
 import (
+	"fmt"
+	"os"
 	"testing"
 
+	"github.com/gluster/glusterd2/context"
 	"github.com/gluster/glusterd2/tests"
+	"github.com/pborman/uuid"
 )
 
+func init() {
+	context.Init()
```

Comment:
The reason of initing the context is to initialize the store as volume create uses the store. Without initing the context the tests fail.